projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -577,6 +595,49 @@ private static function render_error_wrapper() {
 		return $html;
 	}
 
+	/**
+	 * Renders the success wrapper after a form is submitted without reloading the page.
+	 *
+	 * @param Contact_Form $form - the contact form.
+	 *
+	 * @return string HTML string for the success wrapper.
+	 */
+	private static function render_ajax_success_wrapper( $form ) {
+		$html  = '<div class="contact-form-submission contact-form-ajax-submission" data-wp-class--is-submitted="state.hasSubmitted"';
+		$html .= '<p class="go-back-message"> <a class="link" href="#" data-wp-on--click="actions.goBack">' . esc_html__( 'Go back', 'jetpack-forms' ) . '</a> </p>';
```

Comment:
Are the CSS classes these for backwards compatibility, or would it make sense to add `jetpack-` prefix to scope them better?

Reply:
Those are for backwards compatibility, yes. That is almost a copy of the current post-submission screen.